### ISSUE TITLE
⚡ — Pinry

### ISSUE THREAD

**Issue by nomandera** [walkxcode/dashboard-icons#285]
### Type

Icon request

### Info

Please add the logo of the selfhosted Pinterest alternative Pinry

https://github.com/pinry/pinry/

https://github.com/pinry/pinry/tree/master/docs/src/imgs contains some assets but the logo would need extracted from the banner as I cannot find it standalone

e.g.
https://github.com/pinry/pinry/blob/master/docs/src/imgs/logo-dark.png
![image](https://user-images.githubusercontent.com/1133344/220906775-3d15288a-7e45-42b9-9609-e2ad68236dd9.png)

I can have a go at making one if needed although I have no real skills on this area.

![pinry](https://user-images.githubusercontent.com/1133344/221556042-e224ea9c-d5b8-489a-810f-dba51fdf60c7.png)

**Comment by walkxcode** [comment]
Do you happen to have a cropped down version of just the P?

**Comment by nomandera** [comment]
No but leave it with me. I have no skills but this is something I should know.

**Comment by nomandera** [comment]
Does a required specification definition for submission exist? e.g. kB size, px size, required formats, transparency etc

**Comment by walkxcode** [comment]
Not right now, but I should make one. Gonna work on that!

**Comment by nomandera** [comment]
Icon uploaded to opening post for review.

For posterity and because I have no skill in this are and other may benefit:

- Download original icon
- Confirm it is transparent PNG
- In gimp select area to crop. Use manual entry coordinates to make sure it is exactly square. The original was 200px tall so in order to not scale for no real need 200x200px was chosen.
- Export as PNG
- Upload export file to https://tinypng.com/ too apply lossy (sane compression) and strip metadata
- Submit to dashboard-icons

**Comment by walkxcode** [comment]
All issues are temporarily on hold. Please refer to #316 for more info.

**Comment by walkxcode** [comment]
Hey, this icon has been added in the `new-standards` branch and will be merged once it is done!

https://github.com/walkxcode/dashboard-icons/blob/new-standards/png/pinry.png
